projects/storefrontapp/src/app/spartacus/spartacus-b2b-configuration.module.ts

```diff
@@ -30,7 +29,6 @@ if (environment.epdVisualization) {
     ...defaultCmsContentProviders,
     // b2b
     provideConfig(defaultB2bOccConfig),
-    provideConfig(defaultB2BCheckoutConfig),
```

Comment:
[MEDIUM]

Why did we remove this line permanently in this pull request? Wouldn't it break the B2B version of our `storefrontapp`?

Reply:
Reverted.